src/devtools/client/debugger/src/components/Editor/Tabs.tsx

```diff
@@ -2,22 +2,21 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at <http://mozilla.org/MPL/2.0/>. */
 
+import type { Source } from "devtools/client/debugger/src/reducers/sources";
```

Comment:
:speech_balloon: note that this is gonna clash with what I'm doing in my PR.  We can land yours first and I'll rebase to deal with it on my side.

Reply:
Oh thanks. I appreciate that.